feats/app.py

```diff
@@ -168,22 +187,23 @@ def default(self, fn):
         """
         return default(fn)
 
-    def boolean(self, fn):
+    def boolean(self, fn) -> FeatureConditional:
         """
-        Similar to `feature` but operates on a function. Initializes the
+        Similar to `feature` but operates on a boolean function.
```

Comment:
This updated line looks like it kind of garbled the sentence following it. Might be helpful to update that.